src/ontology/mondo-edit.obo

```diff
@@ -481242,7 +481244,7 @@ xref: OMIM:130060 {source="MONDO:equivalentTo"}
 xref: Orphanet:99875 {source="MONDO:equivalentObsolete"}
 xref: UMLS:C3508773 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:854083"}
 is_a: MONDO:0007525 {source="OMIM:130060", source="Orphanet:99875"} ! Ehlers-Danlos syndrome, arthrochalasia type
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/2197 {source="OMIM:130060"} ! COL1A1
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/2197 ! COL1A1
```

Comment:
This is removed since it's a newly added entry into the "exclusion" file https://github.com/monarch-initiative/omim/blob/main/data/exclusions-disease-gene.tsv#L12

Reply:
agreed